python/pyspark/sql/dataframe.py

```diff
@@ -4100,7 +4256,7 @@ def toDF(self, *cols: "ColumnOrName") -> "DataFrame":
         Parameters
         ----------
         cols : str
-            new column names
+            new column names. The length of the list needs to be the same as the number of columns in the initial :class:`DataFrame`
```

Comment:
@Transurgeon mind running `./dev/lint-python` script and fix the line length, etc?

Reply:
yes will do, sorry about that